apps/loan/src/components/PageLlamaMarkets/LendTableFooter.tsx

```diff
@@ -0,0 +1,56 @@
+import Card from '@mui/material/Card'
+import Link from '@mui/material/Link'
+import ArrowOutwardIcon from '@mui/icons-material/ArrowOutward'
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import Grid from '@mui/material/Grid2'
+import Typography from '@mui/material/Typography'
+import type { ReactNode } from 'react'
+import { t } from '@lingui/macro'
+import { SoftLiquidationIcon } from '@/components/PageLlamaMarkets/SoftLiquidationIcon'
+import { CardStackPlusIcon } from '@/components/PageLlamaMarkets/CardStackPlusIcon'
+import { SignIcon } from '@/components/PageLlamaMarkets/SignIcon'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+
+const { Spacing } = SizesAndSpaces
+
+const GridItem = ({ children, title, icon }: { children: ReactNode; title: string; icon: ReactNode }) => (
+  <Grid size={{ tablet: 4, mobile: 12 }}>
+    {icon}
+    <Typography variant="headingXsBold">{title}</Typography>
+    <Typography variant="bodyMRegular">{children}</Typography>
+  </Grid>
+)
+
+export const LendTableFooter = () => (
```

Comment:
Have you considered making the header and button part of the grid with a size 12? Don't forget to set the grid's `rowGap` to `Spacing.sm`.

That way I think you can get rid of the `<Box>` and `width: 100%` in the button below.

Reply:
It's a good idea, it makes the component simpler, but I couldn't get rid of the 100% width